Assets/Scripts/Systems/View3DInputSystem.cs

```diff
@@ -365,7 +365,25 @@ private StateMachine.State SetupFreeMouseState()
 
                 DclSceneManager.SetSelectionRaw(null);
             }
-
+            
+            // Focus the Selected Entity
+            if (Input.GetKeyUp(KeyCode.F) && isMouseOverGameWindow && !CanvasManager.IsAnyInputFieldFocused)
+            {
+                var selectedEntity = DclSceneManager.PrimarySelectedEntity;
+                if (selectedEntity != null)
+                {
+                    var childRenderers = selectedEntity.GetComponentsInChildren<Renderer>();
+                    var magnitude = 10f;
+                    if (childRenderers.Length > 0)
+                    {
+                        var bestRenderer = childRenderers.ElementAtOrDefault(selectedEntity.AllChildCount);
```

Comment:
When the last renderer is very small, the focus will also be quite narrow. Maybe use the larges renderer instead of the last

Reply:
Updated to best bounding box selected by bigger magnitude